### PR TITLE
feat: add build_order method

### PR DESCRIPTION
**Pull request by yogimathius**
This work is in preparation of building an order on successfully matching one with a route

### CODE REVIEW

File: test/models/order_test.rb [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+      },
+      drop_off: {
+        latitude: 34.87433824316913,
+        longitude: -85.08447506395166
+      }
+    }
+    @mock_route = routes(:route1)
+  end
+
+  test "build_order should create a client, package, locations, cargo and order with params" do
```
Comment (yogimathius) [inline test/models/order_test.rb:21]: could add more tests for edge cases, but I think more extensive testing can be considered follow-up work at this point :)
